cmd/ttn-lw-cli/commands/organizations.go

```diff
@@ -74,8 +74,12 @@ var (
 				return err
 			}
 			limit, page, opt, getTotal := withPagination(cmd.Flags())
+			usrID, err := getUserID(cmd.Flags(), nil)
```

Comment:
Ok, but a simple `nil` check would be sufficient, like in the other places. See also my comment in `getUserID` itself.